pyresample/geometry.py

```diff
@@ -1618,7 +1618,7 @@ def get_xy_from_lonlat(self, lon, lat):
 
         :Returns:
```

Comment:
Might as well make this google style (remove the `:` at the start).

Reply:
Done